CHANGELOG.md

```diff
@@ -23,6 +23,22 @@
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+# Unreleased
+
+## Breaking Changes
+
+The following changes only affect implementers which run the conformance test
+suite:
+
+- In preparation for our upcoming [conformance profiles feature][#1709] a new
+  `SupportedFeature` named `HTTPRoute` has been added. Previously there was an
+  implicit assumption in the test suite that implementations supported the
+  `HTTPRoute` API, but this change makes it explicit. All implementations
+  previously running conformance tests **MUST** add the `SupportHTTPRoute`
+  feature to their test suite options if running the tests via the Go library
+  or the command line.
+  (#1890, @shaneutt)
+
```

Comment:
Any reason not just to include this in the release note block? This does seem like a useful idea for incoming major changes, but technically anything can be reverted before we release so I'd hate for people to invest too much in these kinds of notices.

Reply:
Yeah that's fine. Removed and added to PR description.